src/BenchmarksDriver/Program.cs

```diff
@@ -98,6 +98,10 @@ public static int Main(string[] args)
                 "Runtime file attachment. Format is 'path[;destination]', e.g., " +
                 "\"--runtimeFile c:\\build\\System.Net.Security.dll\"",
                 CommandOptionType.MultipleValue);
+            var collectTraceOption = app.Option("--collect-trace",
```

Comment:
I'd like to merge these two arguments, but I don't know how to make the value optional.